parser/src/lib.rs

```diff
@@ -161,6 +161,7 @@ struct Param {
 /// The log level
 #[derive(Clone, Copy, Debug, PartialEq, PartialOrd)]
 pub enum Level {
+    None,
```

Comment:
We could also call it `Always` , which carries the intent that it will be always logged. This is just a suggestion :D

Reply:
Atm this is better than `None`.

Reply:
The level variant was renamed to `Level::Println.`